src/test/java/com/griddynamics/qa/CalculatorTest.java

```diff
@@ -0,0 +1,44 @@
+package com.griddynamics.qa;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import ru.yandex.qatools.allure.annotations.Features;
+import ru.yandex.qatools.allure.annotations.Stories;
+
+import java.lang.reflect.Method;
+
+
+@Features("Calculator testing")
+public class CalculatorTest extends AbstractTest {
+    @DataProvider
+    public Object[][] DataProvider(Method m) {
```

Comment:
What is the reason of the input parameter ? Looks like you don't use it so it can be eliminated.